### DISCUSSION TITLE
滤镜处理图片输出

### DISCUSSION THREAD

**Discussion by ccccxxx** [hooke007/mpv_PlayKit#78] [Q&A 问答]
有什么好的方法用滤镜处理图片并输出吗？
我现在使用的是MPV.NET，先设置图片显示时间无限，然后调滤镜，再用内置截图输出。
我看见其他MPV套壳软件有滤镜处理并输出图片的功能，实现原理是怎样的？

如果要处理很多图片的话，是否能写个脚本来控制台调用MPV处理呢？

**Comment by hooke007** [comment]
> 我看见其他MPV套壳软件有滤镜处理并输出图片的功能

比如？我并不知道有这种软件

> 如果要处理很多图片的话

不是mpv独占滤镜的话直接用ffmpeg更省事。。。。

**Reply by ccccxxx** [reply]
> 
从搜播放器到搜滤镜，终于又翻到了。
原来它不是播放器😂，怪不得当时没下。
https://github.com/AaronFeng753/Waifu2x-Extension-GUI/

**Reply by hooke007** [reply]
链接里的这个项目和mpv没有任何关系。。。

**Reply by ccccxxx** [reply]
> 
> 不是mpv独占滤镜的话直接用ffmpeg更省事。。。。

ffmpeg没用过，能给个教程或者直接告诉我图片批处理加滤镜的语句吗?

**Reply by ccccxxx** [reply]
> 链接里的这个项目和mpv没有任何关系。。。

是的，记模糊了。😓

**Reply by hooke007** [reply]
我不涉猎ffmpeg这方面
我见过的批量用法都是含ffmpeg命令的bat脚本
